### ISSUE TITLE
Runner support?

### ISSUE THREAD

**Issue by andyjakubowski** [exevil/Keys-For-Sketch#18]
First off: loving your plugin! Great idea.

I use Runner to do just about everything in Sketch, and having Keys integrated with that would be awesome. What do you think?

It should be pretty straightforward to set it up: http://sketchrunner.com/developers.html

**Comment by exevil** [comment]
Hey there and thanks for the feedback!

What kind of Keys functionality you'd like to access through Runner, for instance? As for me, there're no **instant** actions in Keys right now. A user just defining some custom shortcuts once and then continue to using Sketch as usual.

The second point is about the straightforward integration: Runner requires all actions to be defined as menu commands, but Keys intentionally doesn't have its own menu so there's no space to define Runner commands there.

Am I missing something? :)

**Comment by andyjakubowski** [comment]
You’re right, I jumped the gun on this one.

I tried to install Keys through the Install tab of Runner, and I was missing _that_ part. But there doesn’t seem to be anything _else_ that could be done via Runner, aside from installing and updating the plugin.

I suppose being able to install Keys right through Runner would be cool, but it’s not a big deal to just download the plugin manually.

Thanks!
